aten/src/ATen/native/cuda/ReflectionPad.cu

```diff
@@ -253,9 +253,271 @@ __global__ void reflection_pad3d_backward_out_kernel(
       });
 }
 
-void reflection_pad2d_out_template(
-    Tensor &output, const Tensor &input_, IntArrayRef padding) {
+// void reflection_pad2d_out_template(
```

Comment:
Hey @Dbhasin1 I see some commented code / failing tests - let me know when you want a review / any help!